src/policykit1.rs

```diff
@@ -207,20 +208,18 @@ impl Subject {
     ///
     /// * `message_header` - The header of the message which caused an authentication to be
     ///   necessary.
-    pub fn new_for_message_header(
-        message_header: &zbus::MessageHeader<'_>,
-    ) -> Result<Self, Error> {
+    pub fn new_for_message_header(message_header: &zbus::MessageHeader<'_>) -> Result<Self, Error> {
         let mut subject_details = HashMap::new();
         match message_header.sender() {
-            Ok(Some(sender)) => {
-                subject_details.insert("name".to_string(), Value::from(sender.clone()).into());
+            Some(sender) => {
+                subject_details.insert(
+                    "name".to_string(),
+                    OwnedUniqueName::from(sender.clone()).try_into().unwrap(),
```

Comment:
a better idea to do that? ideally sender.into() should work I guess.

Reply:
> ideally sender.into() should work I guess.

Yeah, probably. I can't think of what conversions are missing off the top of my head though. This can be improved later on.